hydra-cluster/test/Test/CardanoNodeSpec.hs

```diff
@@ -52,13 +53,9 @@ spec = do
         slot2 `shouldSatisfy` (> slot1)
 
     describe "findRunningCardanoNode" $ do
-      it "returns Nothing on non-matching network" $ \(tr, tmp) -> do
-        withCardanoNodeOnKnownNetwork tr tmp Preview $ \_ -> do
-          findRunningCardanoNode tmp Preproduction `shouldReturn` Nothing
+      it "returns Nothing on non-matching network" $ const $ pendingWith "No other valid network to test against."
```

Comment:
Must: not disable this test

This can be kept by using `Sanchonet` instead of `Preview`

Reply:
:face_exhaling: .. I'll do this then.

Reply:
Done in https://github.com/input-output-hk/hydra/commit/55d55dbdd031639dfa19cc57d4e0c1eeb1af2b8d